### PR TITLE
Enable MINIMAL_RUNTIME + pthreads + node

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

**Comment by sbc100** [comment]
While working a major change to our pthread code (#21701) I realized we didn't yet have any coverage to pthreads + MINIMAL_RUNTIME under node.

File: src/library_pthread.js [hunk 1]
```diff
@@ -441,7 +441,7 @@ var LibraryPThread = {
     allocateUnusedWorker() {
       var worker;
 #if MINIMAL_RUNTIME
-      var pthreadMainJs = Module['worker'];
+      var pthreadMainJs = Module['worker'] || './{{{ PTHREAD_WORKER_FILE }}}';
```
Comment (kripken) [inline src/library_pthread.js:444]: Do we need both here?

**Review by kripken — COMMENTED** [review]
lgtm % question

File: src/library_pthread.js (hunk 1 above)
Comment (sbc100) [reply src/library_pthread.js:444]: I'm honestly not sure.. its possible its being used I suppose?

All this code is hopefully going away soon with #21701 anyway..

Comment (kripken) [reply src/library_pthread.js:444]: Did you add this to fix the test? I'm just trying to understand why this change appears here.

Comment (sbc100) [reply src/library_pthread.js:444]: I added this to make the tests work yes.  Without this we were depending on the HTML to set this.. which doesn't exist when running a test under node.

Comment (sbc100) [reply src/library_pthread.js:444]: (See the code I removed from the HTML file)

Comment (kripken) [reply src/library_pthread.js:444]: I see, thanks, I didn't connect this to that other place. Makes sense now.

**Review by kripken — APPROVED** [review]

**Pull request merged**